package.json

```diff
@@ -24,14 +23,21 @@
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject",
-    "lint": "eslint .",
-    "watch": "webpack --watch"
+    "lint": "eslint ./src",
+    "watch": "webpack --watch",
+    "precommit": "lint-staged"
   },
   "devDependencies": {
-    "eslint": "^4.6.1",
-    "eslint-config-airbnb": "^14.1.0",
-    "eslint-plugin-import": "^2.7.0",
-    "eslint-plugin-react": "^7.3.0"
+    "eslint": "^4.11.1",
```

Comment:
@laurelherting After you merge this PR, you'll need to check out master, git pull, checkout a new feature branch, then do a `yarn`